src/test/java/org/takes/rs/RsWithCookieTest.java

```diff
@@ -69,23 +69,26 @@ public void addsCookieToResponse() throws IOException {
      */
     @Test
     public void addsMultipleCookies() throws IOException {
+        final String qux = "qux";
+        final String bar = "bar";
+        final String value = "value?";
+        final String path = "Path=/qux";
         MatcherAssert.assertThat(
             new RsPrint(
                 new RsWithCookie(
                     new RsWithCookie(
                         new RsEmpty(),
-                        "foo", "works?", "Path=/"
+                        qux,
+                        value,
+                        path
                     ),
-                    "bar", "worksToo?", "Path=/2nd/path/"
+                    bar, "worksToo?", "Path=/2nd/path/"
                 )
             ).print(),
             Matchers.equalTo(
-                Joiner.on("\r\n").join(
-                    "HTTP/1.1 200 OK",
-                    "Set-Cookie: foo=works?;Path=/;",
-                    "Set-Cookie: bar=worksToo?;Path=/2nd/path/;",
-                    "",
-                    ""
+                joiner(
+                    formatCookie(qux, value, path),
```

Comment:
@dalifreire let's call this using the full path `RsWithCookieTest.formatCookie` and do the same for the `joiner` method calls.